crates/adapters/src/transport/datagen.rs

```diff
@@ -1377,12 +1411,24 @@ transport:
 
     #[test]
     fn test_values() {
-        let config_str = r#"
-stream: test_input
-transport:
-    name: datagen
-    config:
-        plan: [ { limit: 4, fields: { "id": { values: [99, 100, 101] } } } ]
+        let config_str = r#"{
+  stream: "test_input",
```

Comment:
In JSON, keys must also be quoted to be valid, e.g., `"stream"` here.

Reply:
Does `serde_json` not throw an error on these tests?

Reply:
Ah yes, it does in the CI: https://github.com/feldera/feldera/actions/runs/10399400940/job/28798214792

Reply:
Yes I haven't converted this, will try to use serde_json5 for everything